finance-tracker/app/views/devise/shared/_links.html.erb

```diff
@@ -0,0 +1,27 @@
+<div class="form-group">
+  <%- if controller_name != 'sessions' %>
+    <%= link_to t(".sign_in"), new_session_path(resource_name) %><br />
+  <% end -%>
```

Comment:
Why use `-%>` syntax?

Reply:
From what I research, it is to remove the next character if it is a newline `\n`, but then after deleting this, I see no difference in views. Are there any other reasons why they use this syntax?

Reply:
to remove the `\n`

Reply:
but then I dont see any `\n` in the code, is it okay to remove this?